src/charm.py

```diff
@@ -63,6 +62,8 @@
 
 PROMETHEUS_CONFIG = "/etc/prometheus/prometheus.yml"
 RULES_DIR = "/etc/prometheus/rules"
+CONFIG_HASH_PATH = "/etc/prometheus/config.hash"
+ALERTS_HASH_PATH = "/etc/prometheus/alerts.hash"
```

Comment:
File extension should likely be `sha256` to conform to common practices.

Reply:
Being specific here with "sha256" is decoupled from the choice of hashing function below, so we would need to remember to update one with the other.
Changed in next commit.